torch/fx/passes/backends/nvfuser/operator_support.py

```diff
@@ -0,0 +1,180 @@
+            "torch.ops.aten.addcmul": None,
+            "torch.ops.aten.native_dropout": None,
+            "torch.ops.aten.dropout": None,
+            "torch.ops.aten.native_dropout_backward": None,
+            "torch.ops.aten.instance_norm": None,
+            "torch.ops.aten._batch_norm_impl_index": None,
+            "torch.ops.aten.native_batch_norm": None,
+            "torch.ops.aten.batch_norm": None,
+            "torch.ops.aten.cudnn_batch_norm": None,
+            "torch.ops.aten._batch_norm_impl_index_backward": None,
+            "torch.ops.aten.native_batch_norm_backward": None,
+            "torch.ops.aten.native_layer_norm": None,
+            "torch.ops.aten.layer_norm": None,
+            "torch.ops.aten.native_layer_norm_backward": None,
+            "torch.ops.aten.softmax.int": None,
+            "torch.ops.aten.log_softmax.int": None,
+            "torch.ops.aten._softmax": None,
+            "torch.ops.aten._log_softmax_backward_data": None,
+            "torch.ops.aten._softmax_backward_data": None,
+            "torch.ops.aten.var.dim": None,
+            "torch.ops.aten.std.dim": None,
+            "torch.ops.aten.sum.dim_IntList": None,
+            "torch.ops.aten.mean.dim": None,
+            "torch.ops.aten._grad_sum_to_size": None,
+            "torch.ops.aten.sum_to_size": None,
+            "torch.ops.aten._autocast_to_reduced_precision": None,
+            "torch.ops.aten._autocast_to_full_precision": None,
+            "torch.ops.aten.to.dtype": None,
+            "torch.ops.aten.type_as": None,
+            "torch.ops.aten.linear": None,
+            "torch.ops.aten.gelu": None,
+            "torch.ops.aten.gelu_backward": None,
+            "torch.ops.aten.tanh_backward": None,
+            "torch.ops.aten.amax": None,
+            "torch.ops.aten.amin": None,
+            "torch.ops.aten.reshape": None,
+            "torch.ops.aten.view": None,
+            "torch.ops.aten.flatten.using_ints": None,
+
+            # ===============================================================
+            # call_function aten: inplace variants
+            # ===============================================================
+            # These nodes shouldn't show up, the functionalization pass should have removed inplace ops
+            "torch.ops.aten.add_": None,
+            "torch.ops.aten.relu_": None,
+
+            # ===============================================================
+            # call_function builtins and operator
+            # ===============================================================
+            "getattr": None,
+            #     "_operator.add": None,
+            #     "_operator.div": None,
+            "_operator.getitem": None,
+            #     "_operator.mul": None,
+            #     "_operator.sub": None,
+            #     "_operator.truediv": None,
+        }
+
+        prim_nvfuser_ops = set(torch._prims.__all__).intersection(dir(fd.Ops))   # type: ignore[attr-defined]
```

Comment:
https://github.com/pytorch/pytorch/pull/79439#discussion_r897937474

Reply:
removed prim ops from this to avoid confusion